src/components/ChatTile.vue

```diff
@@ -69,9 +73,13 @@ export default {
   },
   beforeCreate() {
     const threshold = 0.9;
-    load(threshold).then(model => {
-      this.model = Object.freeze(model);
-    });
+    load(threshold)
+      .then((model) => {
+        this.model = Object.freeze(model);
+      })
+      .catch((e) => {
```

Comment:
Just adding some error logging in case something goes wrong, for easier debugging should that happen on the user's end!